.github/pull_request_template.md

```diff
@@ -1,19 +1,25 @@
-## Done
+### Done
 
-[List of work items including drive-bys]
+- [Add clear and detailed changes description]
 
-## Issue / Card
+---
 
-Fixes #
-
-## QA
+### How to QA
 
-- Pull the branch
-- Run the site using the dotrun snap with `$ dotrun`
-- View the site locally in your web browser at: http://0.0.0.0:8004/
+- Visit the site in your browser at: [Replace with demo link]
+- To run the project locally, please refer to the [Readme](https://github.com/canonical-web-and-design/snapcraft.io/blob/master/README.md#local-development) file for more information
+- If your demo doesn't work, read more about the [demoservice](https://discourse.ubuntu.com/t/demoservice/16862)
```

Comment:
What about moving this as help (smaller) text under the first bullet point about visiting the demo?